inspirehep/modules/records/mappings/records/journals.json

```diff
@@ -204,6 +204,10 @@
                     ],
                     "tokenizer": "standard",
                     "type": "custom"
+                },
+                "lowercase_analyzer": {
+                    "filter": "lowercase",
+                    "tokenizer": "keyword"
```

Comment:
Question: since you ignore case in matching collection in ES, are you anyway then presenting them in the HTML and URLs in their canonical form or you are reusing the case that was provided as input by the user?